tfx/components/example_validator/executor.py

```diff
@@ -112,13 +121,18 @@ def _Validate(self, inputs: Dict[Text, Any], outputs: Dict[Text,
           validation.
         - (Optional) labels.EXTERNAL_CONFIG_VERSION: the version number of
           external config file.
+        - (Optional) labels.TRAINING_STATISTICS: Training statistics to detect
+          skew against
       outputs: A dictionary of labeled output values, including:
           - labels.SCHEMA_DIFF_PATH: the path to write the schema diff to
     """
     schema = value_utils.GetSoleValue(inputs, labels.SCHEMA)
     stats = value_utils.GetSoleValue(inputs, labels.STATS)
+    serving_statistics = value_utils.GetSoleValue(
```

Comment:
Is this training or serving statistics? If the mismatch here is intentional, add a comment here to explain.

Reply:
The mismatch is intentional, I added a comment to explain.